ocaml/idl/datamodel.ml

```diff
@@ -9209,6 +9211,61 @@ let alert =
     ()
 *)
 
+
+(** network sriov **)
+module Network_sriov = struct
+  let lifecycle = [Published, rel_kolkata, ""]
+
+  let configuration_mode = Enum ("configuration_mode",
+    [
+      "sysfs", "Configure network sriov by sysfs, do not need reboot";
+      "modprobe", "Configure network sriov by modbrope, need reboot";
+      "unknown", "Unknown mode";
+    ])
+
+  let create = call
+      ~name:"create"
+      ~doc:"Create a network-sriov based on the specific PIF, it will automatically create a logical PIF to connect the specific network."
+      ~params:[Ref _pif, "pif", "PIF on which to enable SRIOV";
+               Ref _network, "network", "Network to link SRIOV"]
+      ~result:(Ref _network_sriov, "The reference of the created SRIOV object")
```

Comment:
It's a `network_sriov` object.

Reply:
No, it's a reference of the `network_sriov` object as `vlan_create` and `tunnel_create` does.

Reply:
I meant `SRIOV` --> `network_sriov`.

Reply:
haha. ;-)

Reply:
updated.